tests/integration/targets/azure_rm_resource/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: Prepare random number
+  set_fact:
```

Comment:
pls add test to cover subresource

Reply:
ok

Reply:
i didn't want to make any changes to azure_rm_common.py. but now i think it should be a good idea to have azure_rm_common_rest or sth like that

Reply:
i created another common file, so didn't have to touch azure_rm_common at this point

Reply:
how about name it as azure_rm_rest_common.py